libs/components/config/src/lib/params.ts

```diff
@@ -178,44 +179,67 @@ export class SkyAppRuntimeConfigParams {
 
   /**
    * Adds the current params to the supplied link URL.
+   * @param url The url to update
+   * @param queryParams Optional query parameters to include in the constructed url
    */
-  public getLinkUrl(url: string): string {
+  public getLinkUrl(url: string, queryParams?: Params | null): string {
```

Comment:
Instead of using `Params` from `@angular/router` (and requiring a new peer dependency in `libs/components/config/package.json`, which would be considered a breaking change), I would just inline a static type:
```
queryParams?: Record<string, ...> | null
```

Reply:
The Params equivalent is:

```
// eslint-disable-next-line @typescript-eslint/no-explicit-any
queryParams?: Record<string, any>
```

I've updated the PR.